### PR TITLE
[43] Update README.md

### PR DESCRIPTION
**Pull request by trel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -437,22 +439,22 @@ This is required, as the Authorization Code Grant needs to be redirected back to
...
 Currently, the server looks for the custom claim in the ID Token, which is specified in the `irods_user_claim` parameter.
 This serves as the mapping mechanism for an OIDC User to an iRODS User.
 
 A user who authenticates but does not have the claim specified in `irods_user_claim` mapped in their account
-will not have access to the API.
-A HTTP 400 Bad Request status code will be returned if the claim specified in `irods_user_claim` is not found.
+will not have access to the API.  A HTTP 400 Bad Request status code will be returned if the claim specified in `irods_user_claim` is not found.
 
 #### Supported Grants
 Currently, the HTTP API server supports the following two grants:
```
Comment (korydraughn) [inline README.md:450]: Missed an empty line separator.

File: README.md [hunk 2]
```diff
@@ -437,22 +439,22 @@ This is required, as the Authorization Code Grant needs to be redirected back to
...
 
 #### Add your specified `irods_user_claim` to the user's claims
+
 Currently, the server looks for the custom claim in the ID Token, which is specified in the `irods_user_claim` parameter.
 This serves as the mapping mechanism for an OIDC User to an iRODS User.
 
 A user who authenticates but does not have the claim specified in `irods_user_claim` mapped in their account
-will not have access to the API.
-A HTTP 400 Bad Request status code will be returned if the claim specified in `irods_user_claim` is not found.
+will not have access to the API.  A HTTP 400 Bad Request status code will be returned if the claim specified in `irods_user_claim` is not found.
```
Comment (korydraughn) [inline README.md:447]: Collapse two spaces to one after period.

**Review by korydraughn — COMMENTED** [review]
Found a couple things.

File: README.md (hunk 2 above)
Comment (trel) [reply README.md:447]: dubious.   but okay.

**Review by korydraughn — APPROVED** [review]
Squash/Pound when ready.

**Comment by trel** [comment]
#'d

**Pull request merged**